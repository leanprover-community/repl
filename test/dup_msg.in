{"cmd": "def f := 2"}

{"cmd": "#check f", "env": 0}

{"cmd": "#check g", "env": 1}
