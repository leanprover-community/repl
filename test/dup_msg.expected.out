{"env": 0}

{"messages":
 [{"severity": "info",
   "pos": {"line": 1, "column": 0},
   "endPos": {"line": 1, "column": 6},
   "data": "f : Nat"}],
 "env": 1}

{"messages":
 [{"severity": "error",
   "pos": {"line": 1, "column": 7},
   "endPos": {"line": 1, "column": 8},
   "data": "unknown identifier 'g'"}],
 "env": 2}

